scipy/stats/tests/test_distributions.py

```diff
@@ -3388,6 +3389,30 @@ def test_boost_eval_issue_14606(self):
         with pytest.warns(RuntimeWarning):
             stats.beta.ppf(q, a, b)
 
+    @pytest.mark.parametrize('method', [stats.beta.ppf, stats.beta.isf])
+    @pytest.mark.parametrize('a, b', [(1e-310, 12.5), (12.5, 1e-310)])
+    def test_beta_ppf_with_subnormal_a_b(self, method, a, b):
+        # Regression test for gh-17444: beta.ppf(p, a, b) and beta.isf(p, a, b)
+        # would result in a segmentation fault if either a or b was subnormal.
+        p = 0.9
+        # Depending on the version of Boost that we have vendored and
+        # our setting of the Boost double promotion policy, the call
+        # `stats.beta.ppf(p, a, b)` might raise an OverflowError or
+        # return a value.  We'll accept either behavior (and not care about
+        # the value), because our goal here is to verify that the call does
+        # not trigger a segmentation fault.
+        try:
+            stats.beta.ppf(p, a, b)
```

Comment:
Shouldn't this be changed to the `method` `pytest` parameter?

Apart from that, this should be an easy decision to merge--fixes segfault, two core dev approvals, CI passing, small patch with test, etc.

Reply:
Yes indeed it should!  I pushed an update; tests are running again.